osbuild/util/selinux.py

```diff
@@ -56,7 +56,7 @@ def setfiles(spec_file: str, root: str, *paths, exclude_paths: Optional[List[str
                         "-r", root,
                         *exclude_paths_args,
                         spec_file,
-                        f"{root}{path}"],
+                        os.path.join(root, path)],
```

Comment:
was this purely preference or is there actual functional change here?

Reply:
This was a fix for CI:
``` 
root = /run/osbuilt/tree
path = .
....
Error: /run/osbuilt/tree. not found
```